x/mongo/driver/mongocrypt/mongocrypt.go

```diff
@@ -88,6 +88,17 @@ func NewMongoCrypt(opts *options.MongoCryptOptions) (*MongoCrypt, error) {
 		C.mongocrypt_setopt_bypass_query_analysis(wrapped)
 	}
 
+	var keyExpirationMs uint64 = 60_000 // 60,000 ms
+	if opts.KeyExpiration != nil {
+		expirationMs := opts.KeyExpiration.Milliseconds()
+		if expirationMs < 0 {
+			keyExpirationMs = 0
+		} else {
+			keyExpirationMs = uint64(expirationMs)
+		}
```

Comment:
The `Milliseconds` method will truncate values that are less than 1ms to 0, which is interpreted as "never expire". Users might be confused when they put a non-zero expiry value that is interpreted as "never expire".

E.g.
```go
d := 10 * time.Microsecond
fmt.Println(d.Milliseconds())
// Prints "0"
```

We should check for that case and set `expirationMs` to 1.

E.g.
```go
expirationMs := opts.KeyExpiration.Milliseconds()
if expirationMs < 0 {
	keyExpirationMs = 0
} else if expirationMs == 0 && opts.KeyExpiration > 0 {
	// If we get 0ms, but the KeyExpiration duration is non-zero,
	// set keyExpirationMs to 1 so we don't interpret sub-millisecond
	// durations as "never expire".
	keyExpirationMs = 1
} else {
	keyExpirationMs = uint64(expirationMs)
}
```